test/react_client/event_helpers_test.dart

```diff
@@ -182,6 +192,12 @@ main() {
       when(nativeMouseEvent.client).thenReturn(Point(1, 2));
       when(nativeMouseEvent.page).thenReturn(Point(3, 4));
       when(nativeMouseEvent.screen).thenReturn(Point(5, 6));
+      when(nativeMouseEvent.detail).thenReturn(1);
+      when(nativeMouseEvent.view).thenReturn(window);
+      when(nativeMouseEvent.getModifierState('Alt')).thenAnswer((_) {
+        calls.add('getModifierState');
+        return false;
+      });
```

Comment:
Why do only some of the `SyntheticMouseEvent` tests in this file check `movementX` and `movementY`?